packages/main/src/Avatar.hbs

```diff
@@ -1,6 +1,7 @@
 <div
 	class="ui5-avatar-root"
 	tabindex="{{tabindex}}"
+	?disabled="{{disabled}}"
```

Comment:
We don't need that attribute since it's not used anywhere. By default property attribute is attached directly on the custom element and this seems to be redundant duplication.